<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockWidget</class>
 <widget class="QDockWidget" name="DockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ferramentas de Gerência</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="projectTab">
       <attribute name="title">
        <string>Gerência de Projeto</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout"/>
      </widget>
      <widget class="QWidget" name="creationTab">
       <attribute name="title">
        <string>Criação de Projeto</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2"/>
      </widget>
      <widget class="QWidget" name="dangerZoneTab">
       <attribute name="title">
        <string>Zona de Perigo</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3"/>
      </widget>
      <widget class="QWidget" name="metadataTab">
       <attribute name="title">
        <string>Metadata</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
